com.blackmagicdesign.Resolve.yml

```diff
@@ -0,0 +1,186 @@
+id: com.blackmagicdesign.Resolve
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```

Comment:
this runtime will be EOL shortly. 24.08 is the current one.

Reply:
it depends on `libcrypt.so.1` which is not present in newer versions

Reply:
too bad.

Reply:
Building it easy, is a single module, https://github.com/flathub/de.bforartists.Bforartists/blob/8412ecda9a49e852f074c77c6b81285891161462/de.bforartists.Bforartists.yaml#L131-L141